easybuild/easyconfigs/g/GREASY/GREASY-CSCS-19.03-CrayGNU-18.08.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'GREASY'
+version = 'CSCS-19.03'
```

Comment:
Wouldn't it make sense for version to be `19.03_cscs`? I'm not an expert in EB though.

Reply:
This is the release version I did on the repo...

Reply:
This is not an EB thing, the version here is matching the release version in the github repo: https://github.com/eth-cscs/GREASY/releases

If you are not happy with the convention, I think we should first fix that with a new release?

Reply:
👍